kafka-client/src/test/java/dev/responsive/kafka/testutils/Matchers.java

```diff
@@ -1,4 +1,20 @@
-package dev.responsive.kafka.internal.db.testutils;
```

Comment:
felt a bit confusing to have multiple testutils packages in different places, and since this one had only the one class in it I decided to just move this class to the main testutils and get rid of this other testutils package.